.github/workflows/clean-branches.yml

```diff
@@ -0,0 +1,46 @@
+name: Clean Old Branches
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '5 18 * * *'
```

Comment:
Do we need to run it daily?

Reply:
Yes. That means we'll remove a branch over 30 days every day. The result is we'll only keep 30 daily release branches. If we run it weekly, we'll keep 30+7 branches. If one build failed, we'll keep 30+7+7 branches. Since it's a short action, I think it's better to run it every day.